### ISSUE TITLE
Retrieving message history for channels

### ISSUE THREAD

**Issue by alexolog** [EionRobb/purple-discord#214]
In the native client, when I open a channel, I can see the messages that were posted while I was away and  scroll back in the history for as far back as the server persists the messages.  This allows for conversation context.

Can something like that be implemented?

**Comment by chainria** [comment]
Yeah this would really be a cool feature. Especially because the channels ARE logged. The history just cannot be retrieved for whatever reasons are beyond me :(

**Comment by chainria** [comment]
It even gets better than this. I found out that you CAN retrieve the history when using the conversation menu from an active conversation. It's just that you cannot do it from the buddy list itself?

**Comment by alexolog** [comment]
How exactly is it done?
I don't see any such option in the conversation menu.

**Comment by en3r0** [comment]
Would love to see this issue resolved, it is my biggest pain point.
